src/answers-umd.js

```diff
@@ -156,6 +165,24 @@ class Answers {
     globalStorage.set(StorageKeys.SESSIONS_OPT_IN, parsedConfig.sessionTrackingEnabled);
     parsedConfig.noResults && globalStorage.set(StorageKeys.NO_RESULTS_CONFIG, parsedConfig.noResults);
 
+    const masterSwitchApi = statusPage
+      ? new MasterSwitchApi({ apiKey: parsedConfig.apiKey, ...statusPage }, globalStorage)
```

Comment:
do we want to be passing in the api key here still?  I feel like we should take the provided status page as gospel.  If for some wack reason they provide us a statuspage url with an api key param i wouldn't want to override it.

Reply:
The apiKey doesn't really matter here. It's an idiosyncrasy of the ApiRequest class that we include it here. It is not used in determining the actual URL that is hit. We will still hit the one they provide in the statusPage arg.